src/rdkafka_conf.c

```diff
@@ -1702,155 +1751,285 @@ rd_kafka_topic_conf_t *rd_kafka_topic_conf_dup (const rd_kafka_topic_conf_t
 }
 
 rd_kafka_topic_conf_t *rd_kafka_default_topic_conf_dup (rd_kafka_t *rk) {
-        if (rk->rk_conf.topic_conf)
-                return rd_kafka_topic_conf_dup(rk->rk_conf.topic_conf);
-        else
-                return rd_kafka_topic_conf_new();
+        if (rk->rk_conf.topic_conf) {
+            return rd_kafka_topic_conf_dup(rk->rk_conf.topic_conf);
+        }
```

Comment:
coding guidelines, brace placement

Reply:
I will remove it but I would say that 99% of code that i've seen prefers braces. This is to prevent someone from adding a second line to the IF statement and forgetting to add braces. Also it makes the code much easier to read.